From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Sun, 7 Apr 2019 12:51:12 +0200
Subject: Use newly build libs when testing

This exports the LD_LIBRARY_PATH so the newly build libs will be used.
---
 cmake/pcl_targets.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/pcl_targets.cmake b/cmake/pcl_targets.cmake
index 3069634..1769a9a 100644
--- a/cmake/pcl_targets.cmake
+++ b/cmake/pcl_targets.cmake
@@ -390,6 +390,7 @@ macro(PCL_ADD_TEST _name _exename)
     else(${CMAKE_VERSION} VERSION_LESS 2.8.4)
       add_test(NAME ${_name} COMMAND ${_exename} ${PCL_ADD_TEST_ARGUMENTS})
     endif(${CMAKE_VERSION} VERSION_LESS 2.8.4)
+    set_tests_properties(${_name} PROPERTIES ENVIRONMENT "LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/${LIB_INSTALL_DIR}")
 
     add_dependencies(tests ${_exename})
 endmacro(PCL_ADD_TEST)
